mgmt/src/frr/renderer/prefixlist.rs

```diff
@@ -3,40 +3,39 @@
 
 //! Config renderer: prefix list
 
-use crate::frr::renderer::builder::{ConfigBuilder, Render};
-use crate::models::internal::routing::prefixlist::*;
-use std::fmt::Display;
+use crate::frr::renderer::builder::{ConfigBuilder, Render, Rendered};
+use config::internal::routing::prefixlist::*;
 
 /* Impl Display */
-impl Display for PrefixListMatchLen {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+impl Rendered for PrefixListMatchLen {
```

Comment:
I'm ok with this for now, but a better way to do this would be to create a transparent wrapper type around `PrefixListMatchLen` and then implement `Display` for that, then the caller just wraps that.  Same for the other types.  But, not worth changing now.

Reply:
Hum. Why? Isn't the current approach simpler (no extra types) and clearer?

Reply:
This interface doesn't use the formatter and constructs a string directly which adds some overhead.  In this case, I think there is nothing wrong with it per se.  However, the standard way to do this in rust is through a formatter, hence the approach I suggested.  But also, it isn't super important, so I don't think it needs to be changed.

The new code would have looked something like:
```
#[repr(transparent)]
struct Rendered<T>(T)

impl Display for Rendered<PrefixListMatchLen> {
  // Same code as before the move
}
```

Then usage would be identical except instead of doing
```
println!("Value: {v}");
```
You would do:
```
println!("Value: {}", Rendered(v))
```

Reply:
Agreed, we do this in several places. My point in favor of using a very specific trait in that case is that it is for a very particular purpose: making FRR happy. Using such a trait, we minimize the problem of somebody not liking a particular Display and changing it, breaking the integration with FRR.